app/views/components/_publisher-metadata.html.erb

```diff
@@ -0,0 +1,26 @@
+<%
+  published ||= false
+  last_updated ||= false
+  link_to_history ||= false
+  other ||= {}
+  other_has_values = other.values.compact.reject(&:blank?).any?
+  metadata = [published, last_updated]
+%>
+<% if metadata.any? || other_has_values %>
+  <div class="app-c-publisher-metadata">
+    <%= render 'components/published-dates', published: published, last_updated: last_updated, link_to_history: link_to_history %>
+    <% if other_has_values %>
+      <div class="app-c-publisher-metadata__other">
+        <% other.each do |title, values| %>
+          <%
+            values ||= []
+            values = Array(values)
+          %>
+          <% if values.any? %>
+            <p><%= title %>: <%= values.to_sentence.html_safe %></p>
+          <% end %>
+        <% end %>
+      </div>
+    <% end %>
+  </div>
+<% end %>
```

Comment:
Is this replacing the metadata component? If it is, it doesn't look complete yet, is that right? Looking in the component guide it doesn't look like it's properly styled yet, and it doesn't have the toggle functionality of the original component. Sorry, just asking, slightly confused.

Reply:
Kind of...! We're splitting the metadata out into two components: [publisher metadata](https://government-frontend-pr-541.herokuapp.com/component-guide/publisher-metadata) and [important metadata](https://government-frontend-pr-541.herokuapp.com/component-guide/important-metadata). 

I think we have a story in our backlog to add progressive disclosure/toggle to the `important metadata` component. I would assume at the moment that this means long metadata content is just all shown to the user, but I haven't seen any instances of long metadata content in testing yet.

Reply:
The example we found when I originally added it was when there's a list of countries - some pages listed pretty much all the countries, which even on desktop was a lot to fit in.

Reply:
@andysellick do you have the link to that example/know what kind of format it was?

Reply:
I can't find the page on the live site but there's an example in the component guide - https://govuk-static.herokuapp.com/component-guide/metadata/extensive_country_list/preview